ui/apps/platform/src/Containers/MainPage/Sidebar/NavigationSidebar.tsx

```diff
@@ -161,31 +177,36 @@ const navDescriptions: NavDescription[] = [
     },
     {
         type: 'parent',
-        title: (navDescriptionsFiltered) =>
-            navDescriptionsFiltered.some(
-                (navDescription) =>
-                    navDescription.type === 'parent' &&
-                    navDescription.key === keyForVulnerabilityManagement1
-            )
-                ? 'Vulnerability Management (2.0)'
-                : 'Vulnerability Management',
+        title: vulnMgmt2GaContentPredicate(
+            'Vulnerability Management (2.0)',
+            'Vulnerability Management'
+        ),
         key: keyForVulnerabilityManagement2,
         children: [
             {
                 type: 'link',
-                content: <NavigationContent variant="TechPreview">Workload CVEs</NavigationContent>,
+                content: vulnMgmt2GaContentPredicate(
+                    <NavigationContent variant="TechPreview">Workload CVEs</NavigationContent>,
+                    'Workload CVEs'
+                ),
```

Comment:
There has not been discussion on which sections should be marked "Tech Preview", as the expectation is that VM 2.0 will go GA in the next release. If that is not the case, we may need to revisit which items here receive the "Tech Preview" badge.

Since Workload CVE is currently tech preview, I left the badge on that item. I also added the badge to the new Platform CVE and Node CVE links as well.